go_modules/spec/fixtures/projects/ignored_go_files_1.17/_ignored.go

```diff
@@ -0,0 +1,6 @@
+// This file is ignored because it starts with an underscore
```

Comment:
These files are copied from the other spec, and aren't really relevant to this particular test, but I thought they didn't hurt to test that the behavior doesn't change in the newer go versions.